packages/ui/libs/ui/core/src/components/layout/PermissionsFeedback/PermissionsFeedback.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { memo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { IPermissionRule } from '@energyweb/origin-ui-utils';
+import {
+  Typography,
+  List,
+  ListItem,
+  ListItemIcon,
+  Checkbox,
+  ListItemText,
+} from '@material-ui/core';
+
+interface IProps {
+  accessRules: IPermissionRule[];
+}
+
+export const PermissionsFeedback = memo(({ accessRules }: IProps) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Typography variant="body1" className="mt-3" gutterBottom>
+        {t('general.requirements.requirementsTitle')}
```

Comment:
We should not have localization in core components. We should pass them as props e.g. title